app/src/main/java/fr/free/nrw/commons/upload/ThumbnailsAdapter.java

```diff
@@ -87,16 +89,21 @@ public void bind(int position) {
             background.setImageURI(Uri.fromFile(new File(String.valueOf(uri))));
 
             if (position == callback.getCurrentSelectedFilePosition()) {
+                GradientDrawable border = new GradientDrawable();
+                border.setShape(GradientDrawable.RECTANGLE);
+                border.setStroke(10, context.getResources().getColor(R.color.primaryColor));
                 rlContainer.setEnabled(true);
                 rlContainer.setClickable(true);
                 rlContainer.setAlpha(1.0f);
+                rlContainer.setForeground(border);
```

Comment:
Looks like we can't use `setForeground` for this. As Android Studio says it's supported only from API 23 and above but our app supports API 19 and above. As a matter of fact, the app crashes for me when I try to share multiple images to the app. I use an Android 5.1.1 device.

Reply:
Fixed the API issue ,
I would like to apologize for the chaos I created, but actually I typed some wrong command and reverting changes wasn't helping.
Now regarding the issue : I think everything is working fine and borders are also over images now, I have also added some padding pixels in cardview.

DOUBT : like my last branch had 5-6 commits. So After each commit I am working on the updated code right? So if did something wrong in the past commit and I didn't remove the extra variables, libraries, code etc.( that i pushed along with earlier commit) in the corrected commit, so while merging do maintainers get the option to select the commits they want to accept? 
In my last branch I was deleting those earlier commits so that I could push the corrected code but somehow while moving the HEAD~ I used the wrong command. I again apologize for the problems I caused. It won't happen again.